programs/perpetuals/src/state/custody.rs

```diff
@@ -265,6 +259,9 @@ impl Custody {
     }
 
     pub fn lock_funds(&mut self, amount: u64) -> Result<()> {
+        if self.is_virtual {
+            return Ok(());
```

Comment:
Should ideally throw to avoid conflict with later design

Reply:
updated